zerver/lib/actions.py

```diff
@@ -2070,7 +2070,9 @@ def check_message(sender: UserProfile, client: Client, addressee: Addressee,
         except ValidationError as e:
             assert isinstance(e.messages[0], str)
             raise JsonableError(e.messages[0])
-    else:
+    else:  # nocoverage
+        # This is defensive code--Addressee already validates
+        # the message type.
         raise JsonableError(_("Invalid message type"))
```

Comment:
I think changing this to `AssertionError` would be appropriate here, if it's no longer a possible code flow.